### PR TITLE
Add pydv

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for pydv, a data visualisation package used in LIGO

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, this package is ready for review. It's marked as `noarch: python`, but the dependencies are not available for windows at this time, hence the `win_64` azure failure.

**Comment by duncanmmacleod** [comment]
@jrollins, can you post a comment to indicate that you are happy to be named as a maintainer of this package, thanks?

File: recipes/pydv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - pydv.const
+    - pydv.plotting
+    - pydv.util
+  commands:
+    - pydv --help
+
+about:
+  home: https://git.ligo.org/cds/pydv/
+  dev_url: https://git.ligo.org/cds/pydv/
+  license: GPLv3+
```
Comment (chrisburr) [inline recipes/pydv/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```

Comment (duncanmmacleod) [reply recipes/pydv/meta.yaml]: @chrisburr, according to https://spdx.org/licenses/GPL-3.0.html the actual identifier should probably be `GPL-3.0-or-later`, I am happy to change it if the recommendation is to use SPDX identifiers in this field?

Comment (chrisburr) [reply recipes/pydv/meta.yaml]: So far as I know the SPDX identifiers are prefered: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L60

(Thanks for the correction with `-or-later`, I should use that more often)

**Comment by jrollins** [comment]
@duncanmmacleod yes I'm fine being listed as the maintainer.

**Pull request merged**